README.ja.rdoc

```diff
@@ -76,10 +76,9 @@
 == ライセンス
 
   このパッケージに付属するファイルの著作権は青木峰郎が保持します。
```

Comment:
You _must_ get @aamine's approval before merging this.  As you can see he has the copyright.  You must not bend human rights of someone else.

Reply:
I already to get approval from aamine. https://github.com/ruby/racc/pull/105#issuecomment-490491506

Is it not enough?

Reply:
Yes, definitely.  I was not aware of that comment.